docs/sources/auth/enhanced_ldap.md

```diff
@@ -41,3 +41,11 @@ a user as member of a team and it will not be removed when the user signs in. Th
 5. Click on `Add group` button to save.
 
 <div class="clearfix"></div>
+
+## Active LDAP Synchronization
+
+With the open source version of grafana LDAP users will be sync only during the log in process.
```

Comment:
Change to the something like the following? `With the open source version of Grafana, user data from LDAP will be synchronized only during the login process when authenticating using LDAP.`

Reply:
Done